.github/workflows/runtimes-matrix.json

```diff
@@ -31,7 +31,7 @@
     "path": "system-parachains/asset-hubs/asset-hub-kusama",
     "uri": "wss://kusama-asset-hub-rpc.polkadot.io:443",
     "is_relay": false,
-    "blocktime": 6000,
+    "blocktime": 12000,
```

Comment:
Wondering about this change, not familiar with this matrix. Should it be the block time or slot duration here? Because blocktime will be 6s.